Background Image name was changed from background.png to template.png

Signed-off-by: Saul Wold <sgw@linux.intel.com>

Index: matchbox-sato/Sato/matchbox2/Makefile.am
===================================================================
--- matchbox-sato.orig/Sato/matchbox2/Makefile.am	2010-12-29 13:08:43.000000000 -0800
+++ matchbox-sato/Sato/matchbox2/Makefile.am	2010-12-29 22:14:45.690428881 -0800
@@ -1,4 +1,4 @@
-pngs = background.png
+pngs = template.png
 
 themesdir = $(datadir)/themes/Sato/matchbox2
 themes_DATA = theme.xml $(pngs)
